digitsGroup_test.go

```diff
@@ -0,0 +1,76 @@
+package digits
+
+type stimulus struct {
+	value               string
+	groupingCharRune    rune
+	groupingCharStr     string
+	ifSigFig            bool
+	fractionalPrecision Decimals
+}
+type output struct {
+	triplified string
+	final      string
+}
+type testCase struct {
+	stimulus stimulus
+	output   output
+}
+
+var testCases = []*testCase{
+	{
+		stimulus: stimulus{
+			value:               "45121000000000.00",
+			groupingCharRune:    ',',
+			groupingCharStr:     ",",
+			ifSigFig:            true,
+			fractionalPrecision: PreserveUpToHundredth,
+		},
+		output: output{
+			triplified: "45,121,000,000,000",
+			final:      "45,121,000,000,000.00",
+		},
+	},
+	{
+		stimulus: stimulus{
+			value:               "80800000.00",
+			groupingCharRune:    ',',
+			groupingCharStr:     ",",
+			ifSigFig:            true,
+			fractionalPrecision: PreserveUpToHundredth,
+		},
+		output: output{
+			triplified: "80,800,000",
+			final:      "80,800,000.00",
+		},
+	},
+}
+
+/*
+func Test_DigitGroup(t *testing.T) {
+	for _, testCase := range testCases {
+		sut := DigitGroup(Exact, testCase.stimulus.value, testCase.stimulus.groupingCharRune, testCase.stimulus.fractionalPrecision, testCase.stimulus.ifSigFig)
+		if sut != testCase.output.final {
+			t.Fail()
+		}
+	}
+}
+
+func Test_Triplefy(t *testing.T) {
+	for _, testCase := range testCases {
+		sut := triplefy(testCase.stimulus.value, testCase.stimulus.groupingCharStr, testCase.stimulus.ifSigFig)
+		if sut != testCase.output.triplified {
+			t.Fail()
+		}
+	}
+}
+
+func Test_ZeroTriplefy(t *testing.T) {
+	//TODO: flesh this out in stimulus
+	for _, testCase := range testCases {
+		sut := zeroTriplefy(testCase.stimulus.value, testCase.stimulus.groupingCharStr)
+		if sut != testCase.output.triplified {
+			t.Fail()
+		}
+	}
+}
```

Comment:
nit: These test cases probably don't bring much value.  Just delete this commented block.

Reply:
this was mentioned in [38305aa](https://github.com/joshuanario/digits/pull/6/commits/38305aad36b7a8eb933fb8470e1b3b01610d7c20). I can go ahead and remove them, but mostly they were failing given the different direction we took the compute functions. they would be relevant for main in it's current state.

I'll make a commit to remove that test file, and we can always revisit it if needed.

Reply:
[f5c601c](https://github.com/joshuanario/digits/pull/6/commits/f5c601c1871e252d262cdbe3d035880c5030323c)